ml/Config.cc

```diff
@@ -123,6 +128,18 @@ void Config::readMLConfig(void) {
     Cfg.ADWindowRateThreshold = ADWindowRateThreshold;
     Cfg.ADDimensionRateThreshold = ADDimensionRateThreshold;
 
+    Cfg.MaxAnomalyRateInfoTableRows = MaxAnomalyRateInfoTableRows;
+    Cfg.MaxAnomalyRateInfoAge = MaxAnomalyRateInfoAge;
+
     Cfg.SP_HostsToSkip = simple_pattern_create(HostsToSkip.c_str(), NULL, SIMPLE_PATTERN_EXACT);
     Cfg.SP_ChartsToSkip = simple_pattern_create(ChartsToSkip.c_str(), NULL, SIMPLE_PATTERN_EXACT);
+
+    #if 0
+        Cfg.MinTrainSamples = 1 * 60;
+        Cfg.MaxTrainSamples = 4 * 60;
+        Cfg.TrainEvery = 1 * 60;
+
+        Cfg.SP_ChartsToSkip = simple_pattern_create("!system.cpu *", NULL, SIMPLE_PATTERN_EXACT);
+    #endif
```

Comment:
We should delete these because we want to be able to share the PR with others.

Reply:
Removed.